WordPress/WordPressApi/WordPressComRestApi.swift

```diff
@@ -88,8 +94,10 @@ public class WordPressComRestApi: NSObject
                      success: SuccessResponseBlock,
                      failure: FailureReponseBlock) -> NSProgress?
     {
+        let URLString = appendLocaleParameterToURLPathIfNeeded(URLString)
```

Comment:
Wouldn't it be much simpler to add the `locale` value to `parameters` instead of modifying the URL?